github/repos.go

```diff
@@ -213,9 +215,10 @@ func (s *RepositoriesService) ListByOrg(ctx context.Context, org string, opt *Re
 		return nil, nil, err
 	}
 
-	// TODO: remove custom Accept header when license support fully launches
-	req.Header.Set("Accept", mediaTypeLicensesPreview)
+	// TODO: remove custom Accept headers when APIs fully launch.
+	acceptHeader := []string{mediaTypeLicensesPreview, mediaTypeTopicsPreview}
 
+	req.Header.Set("Accept", strings.Join(acceptHeader, ", "))
```

Comment:
Here too. Rename to `acceptHeaders`, no blank line between `acceptHeaders` and `req.Header.Set`, blank line after.